ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-periodics.yaml

```diff
@@ -66,18 +68,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: packet-edge
     ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/variant: edge
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-assisted-installer-agent-master-subsystem-test-periodic
```

Comment:
Seems like we're missing those kind of notifications

Reply:
I went through and put the slack notifications back in